$OpenBSD: patch-doc_reference_texi,v 1.3 2011/07/28 12:23:01 jasper Exp $

commit 2ea3da9da565f0c4d115d79e726635df10d66b9f
Author: Jasper Lievisse Adriaanse <jasper@humppa.nl>
Date:   Thu Jul 28 14:18:33 2011 +0200

    Fix makeinfo(1) warning in reference.texi
    
    doc/reference.texi:746: warning: @strong{Note...} produces a
    spurious cross-reference in Info; reword to avoid that.
    
    https://bugzilla.gnome.org/show_bug.cgi?id=653180

--- doc/reference.texi.orig	Mon Apr  5 15:05:02 2010
+++ doc/reference.texi	Mon Apr  5 15:05:32 2010
@@ -743,7 +743,7 @@ The description above is taken from the manual page of
 filesystem under Linux and is a little bit confusing, so I make this clear
 here. 
 
-@strong{Note for people porting LibGTop to other systems:}
+@strong{A word for people porting LibGTop to other systems:}
 Every operating system has its own idea about the memory usage of a process
 and also system utilities like @code{ps} show different things on different
 systems.
